src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.container');
+const tableBody = document.querySelector('.field').firstElementChild;
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+table.addEventListener('click', (event) => {
```

Comment:
let's attach a separate event listener to each button

Reply:
i already have another solution with separate listeners, this is my very first solution)